src/ansys/motorcad/core/methods/rpc_methods_graphs.py

```diff
@@ -163,6 +170,50 @@ def get_fea_graph_point(self, graph_id, slice_number, point_number, time_step_nu
         params = [{"variant": graph_id}, slice_number, point_number, time_step_number]
         return self.connection.send_and_receive(method, params)
 
+    def get_fea_graph(self, graph_name, slice_number, time_step_number):
+        """Get graph points from a Motor-CAD FEA graph.
+
+        Parameters
+        ----------
+        graph_name : str, int
+            Name (preferred) or ID of the graph. In Motor-CAD, you can
+            select **Help -> Graph Viewer** to see the graph name.
+        slice_number : int
+            Slice number to get x and y coordinate values from.
+        time_step_number : int
+            Time step number to get x and y coordinate values from.
+        Returns
```

Comment:
```suggestion
        
        Returns
```
Blank line is necessary for rendering correctly in the doc.